### DISCUSSION TITLE
The ability to drag down the toolbar + a button to hide it for mobile users.

### DISCUSSION THREAD

**Discussion by dumbuz** [ppy/osu#22905] [Ideas]
I just learned through a startup hint that the toolbar is toggleable. I think a great addition to this would be allowing mobile users to hide the toolbar via a touch button (maybe in one of the corners of the toolbar) and allowing them to drag from the top to get it back. In places like the pause menu, it is impossible to access any menus unless you have a keyboard connected.

**Comment by shiumano** [comment]
Similarly, we might want to display settings when you swipe from the left.

**Reply by Gabixel** [reply]
Someone has already opened an issue to suggest it (and so it's in the catalog of future things)! :D
https://github.com/ppy/osu/issues/6493 - so I think your suggestions are probably related to this old thread
